packages/rust/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://www.rust-lang.org/
+TERMUX_PKG_DESCRIPTION="Systems programming language focused on safety, speed and concurrency"
+TERMUX_PKG_VERSION=1.29.2
+TERMUX_PKG_SRCURL=https://static.rust-lang.org/dist/rustc-$TERMUX_PKG_VERSION-src.tar.gz
+TERMUX_PKG_SHA256=5088e796aa2e47478cdf41e7243fc5443fafab0a7c70a11423e57c80c04167c9
+TERMUX_PKG_DEPENDS="clang, openssl"
+
+termux_step_configure () {
+	termux_setup_cmake
+
+	if [ "$TERMUX_ARCH" = "arm" ]; then
+		TRIPLE="armv7-linux-androideabi"
```

Comment:
Current triple is almost same, only missing `v7`, could be fixed [this way instead](https://github.com/termux/termux-packages/blob/master/packages/ldc/build.sh#L40).